src/main/webapp/index.html

```diff
@@ -81,25 +81,26 @@ <h3 class="panel__section-heading">Mail</h3>
             </a>
             <h3 class="panel__section-heading">Tasks</h3>
           </div>
-          <p class="panel__section-subheading">Selected Tasks:
+          <p class="panel__section-subheading">Selected Tasks: 
```

Comment:
Is this selecting tasks or task lists? This text and the select id imply you are selecting tasks.

Reply:
Agreed, naming seems to be a common cause of confusion for tasks and task lists. Thanks for pointing that out!

Reply:
`RESOLVED`